scene/gui/range.cpp

```diff
@@ -373,6 +376,14 @@ bool Range::is_lesser_allowed() const {
 	return shared->allow_lesser;
 }
 
+void Range::set_allow_non_finite(bool p_allow) {
+	allow_non_finite = p_allow;
+}
+
+bool Range::is_non_finite_allowed() const {
+	return allow_non_finite;
+}
```

Comment:
Perhaps it makes sense to expose this as a property.

Reply:
do you know if there would be any uses for that?

Reply:
Custom editor tools, much more than anything.

I am skeptical on exposing it myself, as I would rather have at least one individual ask for it, first, so as not to bloat the API. 
_Bbbbbbut_ there were users that did in fact rely on the previous behavior, as well (but only in an editor context, notably)

Reply:
Ok, let's wait for an explicit request.